recipes/datapane/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set name = "datapane" %}
+{% set version = "0.6.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b4daafa9c94fb12f5ba3ecfe74d51256023a97223a5ef9db88fc9c3c7078ad8d
+  
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - datapane = datapane.client.__main__:main
+    - dp-runner = datapane.runner.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - poetry
+  run:
+    - python
+    - dataclasses
+    - altair >=4.0.0,<5.0.0
+    - bokeh >=2.0.0,<3.0.0
+    - click >=7.0.0,<8.0.0
+    - click-spinner >=0.1.8,<0.2.0
+    - colorlog >=4.1.0,<5.0.0
+    - dacite >=1.5.0,<2.0.0
+    - flit-core >=2.3.0,<2.4.0
+    - folium >=0.11.0,<0.12.0
+    - importlib_resources >=3.0.0,<4.0.0
+    - jinja2 >=2.11.1,<3.0.0
+    - jsonschema >=3.2.0,<4.0.0
+    - matplotlib >=3.1.0,<4.0.0
```

If you don't need the qt frontend ([more details](https://conda-forge.org/docs/maintainer/knowledge_base.html#matplotlib)):
```suggestion
    - matplotlib-base >=3.1.0,<4.0.0
```

---

Got it - will update and repush